src/app/charity/charity.component.html

```diff
@@ -10,10 +10,30 @@ <h1 *ngIf="campaigns[0]" class="b-rh-1 b-bold">{{ campaigns[0].charity.name }} C
       </p>
     </div>
 
-    <div class="c-grid">
-      <div *ngFor="let campaign of campaigns">
-        <app-campaign-card [campaign]="campaign"></app-campaign-card>
-      </div>
-    </div>
+    <biggive-grid
+        slot="campaign-grid"
+        column-count="3"
+        *ngIf="campaigns"
+      >
+        <div *ngFor="let campaign of campaigns">
+          <biggive-campaign-card
+            id="campaign-{{campaign.id}}"
+            [banner]="campaign.imageUri | optimisedImage:830 | async"
+            [campaignType]="campaign.isMatched ? 'Match Funded' : null"
+            [campaignTitle]="campaign.title"
+            [donateButtonUrl]="'/donate/' + campaign.id"
+            [moreInfoButtonUrl]="'/campaign/' + campaign.id"
+            [organisationName]="campaign.charity.name"
+            primary-figure-label="Match Funds Remaining"
+            [primaryFigureAmount]="campaign.matchFundsRemaining | currency: campaign.currencyCode : 'symbol' : '1.0-0'"
+            [progressBarCounter]="getPercentageRaised(campaign)"
+            secondary-figure-label="Total Funds Raised"
+            [secondaryFigureAmount]="campaign.amountRaised | currency: campaign.currencyCode : 'symbol' : '1.0-0'"
+            [isFutureCampaign]="isInFuture(campaign)"
+            [isPastCampaign]="isInPast(campaign)"
+            [datetime]="getRelevantDateAsStr(campaign)">
```

Comment:
Is this code just moved from a now deleted component? If so there's no need to review the details - if not @NoelLH can you let me know which parts are different? I'm not sure if there's any easy way to see that.

Reply:
@bdsl Not exactly, the new version is adapted from the meta-campaign grid – this allows us to remove the old design card from Donate entirely as this was the last page using it. You can see the before and after visually in Jira.